Cython/Compiler/Builtin.py

```diff
@@ -226,7 +226,16 @@ def declare_in_type(self, self_type):
     #('reduce',    "",     "",      ""),
     BuiltinFunction('reload',     "O",    "O",     "PyImport_ReloadModule"),
     BuiltinFunction('repr',       "O",    "O",     "PyObject_Repr", builtin_return_type='str'),
-    #('round',     "",     "",      ""),
+    BuiltinFunction('round',        "f",    "f",     "roundf",
+                    is_strict_signature=True, nogil=True),
+    BuiltinFunction('round',        "d",    "d",     "round",
+                    is_strict_signature=True, nogil=True),
+    BuiltinFunction('round',        "ld",    "ld",     "roundl",
```

Comment:
The "ld" is not a valid signature description and is actually not used here due to the provided complete function type.
```suggestion
    BuiltinFunction('round',        None,    None,     "roundl",
```

Reply:
I know, I thought I already removed that.